src/components/Tasks/TaskStatus.js

```diff
@@ -19,7 +18,11 @@ const TaskStatus = React.memo(function TaskStatus({ dueDate }) {
   const [refresh, setRefresh] = useState(false)
 
   const now = new Date()
-  const refreshTime = dueDate - now
+
+  const endofTime = dayjs()
+    .endOf('day')
+    .add(1, 'millisecond')
+  const refreshTime = endofTime - now
```

Comment:
Not sure on this; we might want to use `diff(now, 'millisecond')` or convert the dayjs variable into an actual Date (it might implicitly do this, but it feels better to know we're using two native Date objects than one dayjs object and one Date object).

It might also be clearer as `const startOfNextDay = dayjs().add(1, 'day').startOf('day')`